### ISSUE TITLE
Sh1mmer utilility likely did not work

### ISSUE THREAD

**Issue by ben10101010** [CoolElectronics/sh1mmer.me#43]
Everytime I try using an option in the sh1mmer menu, unenrollment, usb unblock, dev mod, etc, it gives me an error saying “Sh1mmer utility likely did not work”. I have tried using the shell to run the commands by themselves, but I have had no luck so far. How can I fix this?

**Comment by TheMemeSniper** [comment]
has your device EVER updated to ≥r111

**Comment by velzie** [comment]
duplicate of #42

**Comment by ben10101010** [comment]
I just downgraded my chromebook to version 108 but sh1mmer still gives me the same error.

**Comment by velzie** [comment]
yes, it's supposed to, read #42

**Comment by ben10101010** [comment]
I disabled dev mode, it gave me the error, then when I went to the menu to get into developer mode, the screen was blocked and did not work.
